### PR TITLE
docs: some fixes and typos for benchmark page

### PR DESCRIPTION
**Pull request by cristianmtr**
- typos
- phrasing
- clarity

### CODE REVIEW

File: docs/advanced/document-store/benchmark.md [hunk 1]
```diff
@@ -10,9 +10,9 @@
 :scale: 0 %
 ```
 
-We create a DocumentArray with one million Documents, and benchmark all supported document stores. This includes classic database and vector database all under the same DocumentArray API:
+We create a DocumentArray with one million Documents and benchmark all supported document stores. This includes classic database and vector database, all under the same DocumentArray API:
 
-* {ref}`"None"<documentarray>`: `DocumentArray()`, namely an in-memory "store". In the sequel, in-memory/memory store refers to this.
+* {ref}`"None"<documentarray>`: `DocumentArray()`, namely an in-memory "store"
```
Comment (alaeddine-13) [inline docs/advanced/document-store/benchmark.md:15]: I guess here the intention is to let users know that we refer to the default usage of `DocumentArray` when we say `in-memory store`

File: docs/advanced/document-store/benchmark.md [hunk 2]
```diff
@@ -28,11 +28,11 @@ We focus on the following tasks:
 5. **Find by condition**: search existing Documents by `.tags` via {meth}`~docarray.array.mixins.find` in the document store by boolean filters, as described in {ref}`find-documentarray`.
 6. **Find by vector**: retrieve existing Documents by `.embedding` via {meth}`~docarray.array.mixins.find`  using  nearest neighbor search or approximate nearest neighbor search, as described in {ref}`match-documentarray`.
 
-We are interested in the single-query performance on the above tasks, which means the tasks 2,3,4,5,6 are evaluated using one Document at a time, repeatedly, and the average number is reported.
+We are interested in the single query performance on the above tasks, which means tasks 2,3,4,5,6 are evaluated using one Document at a time, repeatedly. We report the average number.
 
 ## Benchmark result
 
-The following table summarizes the result; values are smaller the better (except for `Recall@10`). The best performer of each task is highlighted with bold font:
+The following table summarizes the result. Values are smaller the better (except for `Recall@10`). The best performer of each task is highlighted in **bold**:
```
Comment (alaeddine-13) [inline docs/advanced/document-store/benchmark.md:35]:
```suggestion
The following table summarizes the result. The smaller the values, the better (except for `Recall@10`). The best performer of each task is highlighted in **bold**:
```
do you think this is more gramatically correct ?

File: docs/advanced/document-store/benchmark.md [hunk 3]
```diff
@@ -121,73 +121,67 @@ We use `Recall@K` value as an indicator of the search quality. The in-memory and
 The experiments were conducted on a 4.5 Ghz AMD Ryzen Threadripper 3960X 24-Core Processor with Python 3.8.5.
 
 Besides, as Weaviate, Qdrant and ElasticSearch follow a client/server pattern, we set up them with their official 
-docker images in a **single node** configuration, with 40 GB of RAM allocated. That is, only 1 replica and shard is 
+docker images in a **single node** configuration, with 40 GB of RAM allocated. That is, only 1 replica and shard are 
 operated during the benchmarking. We did not opt for a cluster setup because our benchmarks mainly aim to assess the 
-capabilities of a single instance of the backend.
+capabilities of a single instance of the service.
```
Comment (alaeddine-13) [inline docs/advanced/document-store/benchmark.md:126]:
```suggestion
capabilities of a single instance of the server.
```
maybe server is a better terminology ?

File: docs/advanced/document-store/benchmark.md [hunk 4]
```diff
@@ -198,6 +192,6 @@ SQLite store is omitted from the left plot and the in-memory store is omitted fr
 slow and the in-memory is too fast to fit into the figure.
 ```
 
-AnnLite is a good choice when indexing/appending/inserting speed matters more than the speed of finding. Moreover, as AnnLite is a local monolith package that does not follow a client-server design, hence it avoids all network overhead.
+AnnLite is a good choice when indexing/appending/inserting speed matters more than the speed of finding. Moreover, AnnLite is a local monolith package that does not follow a client-server design, so it avoids all network overhead.
```
Comment (alaeddine-13) [inline docs/advanced/document-store/benchmark.md:195]:
```suggestion
AnnLite is a good choice when indexing/appending/inserting speed matters more than the speed of finding. Moreover, AnnLite is a local monolithic package that does not follow a client-server design, so it avoids all network overhead.
```

File: docs/advanced/document-store/benchmark.md (hunk 1 above)
Comment (cristianmtr) [reply docs/advanced/document-store/benchmark.md:15]: Yes, it's said in the first part of the sentence

> * {ref}`"None"<documentarray>`: `DocumentArray()`, namely an in-memory "store"

File: docs/advanced/document-store/benchmark.md (hunk 2 above)
Comment (cristianmtr) [reply docs/advanced/document-store/benchmark.md:35]: Yes

File: docs/advanced/document-store/benchmark.md (hunk 3 above)
Comment (cristianmtr) [reply docs/advanced/document-store/benchmark.md:126]: Sure, either.

**Comment by cristianmtr** [comment]
@alaeddine-13 Check again. Implemented your suggestions

**Review by alaeddine-13 — APPROVED** [review]

**Pull request merged**